lib/yardstick/decorator.rb

```diff
@@ -0,0 +1,55 @@
+module Yardstick
+  # A string decorator for applying unix console codes
+  class Decorator
+    include Concord.new(:color, :mode)
```

Comment:
I've updated this and some other classes here: https://github.com/dkubb/yardstick/commit/18e55d5f0dbc4f454853c9aa6dadb04a0c3ec8e8